google-cloud-logging/src/test/java/com/google/cloud/logging/BaseSystemTest.java

```diff
@@ -468,12 +468,14 @@ public void testLoggingHandler() throws InterruptedException {
     logger.addHandler(handler);
     logger.setLevel(Level.INFO);
     logger.info("Message");
+    String filter =
+            "logName = projects/" + logging().getOptions().getProjectId() + "/logs/" + logName;
```

Comment:
Use LogName now. We had some unfortunate name clash. The `logName` in the code should be "logId", whereas the [doc](https://cloud.google.com/logging/docs/reference/v2/rest/v2/LogEntry) suggests that "logName"  is a fully qualified name prefixed with resourceName.  PTAL.